FlowCrypt/Functionality/Mail Provider/Message Gateway/Imap+send.swift

```diff
@@ -6,9 +6,9 @@ import Foundation
 
 extension Imap: MessageGateway {
     func sendMail(input: MessageGatewayInput, progressHandler: ((Float) -> Void)?) async throws {
-        try await withCheckedThrowingContinuation { [weak smtpSess] (continuation: CheckedContinuation<Void, Error>) in
-            guard let session = smtpSess else {
-                return continuation.resume(returning: ())
```

Comment:
@sosnovsky on master there is an error with submitting key to attester with IMAP usage so it would not reproduce

Reply:
@Kharchevskyi I think for testing new public key submit you need to delete your existing public key at https://flowcrypt.com/attester/
But not sure if it'll affect your previous emails which were using your current public key.

Reply:
If i remove this pub key from attester manually as you suggest flow works. 
I can create a new backup in inbox and submit it. 

But if I remove this backup from web interface and try to setup a new backup I receive the same error.
Do we need to handle it somehow?

Reply:
Yes, it should be better to have alert with such actions for key error
- `Replace public key` - opens  https://flowcrypt.com/attester/update-public-keys
- `Delete public key` - opens https://flowcrypt.com/attester/update-public-keys?intention=delete
- `Cancel` - redirects user back to sign in screen

@tomholub does such error handling looks good for you?

Reply:
@sosnovsky thank you for the suggestion - it's well meaning and a good idea, but in this case unneeded because this is such a fringe situation.

We have nearly zero users using SMTP/IMAP. In the future we may have more, but it will never be nearly enough to to amount of users that we have for Oauth2 due to the difference in convenience. I'll file the suggestion as a separate issue for sometime later in the future.